### ISSUE TITLE
python version requirement for asm2cfg

### ISSUE THREAD

**Issue by gigglesun** [Kazhuu/asm2cfg#31]
I used python3.8 and gdb 12.1, when start gdb, it will have following message, did this tool have some python version requirement?

```
~/anaconda3/bin/gdb_asm2cfg.py", line 32
    from splitting CFG blocks: {self.value_to_string()}'
                                                       ^
SyntaxError: invalid syntax
```
I checked line 32, it is below function:
`
def get_set_string(self):
    return f'Commands savecfg and viewcfg will skip function calls \
        from splitting CFG blocks: {self.value_to_string()}'
`

**Comment by Kazhuu** [comment]
Python 3.6 should be minimum here because of the f-strings. Did you try to install the latest release I made yesterday v0.2.1?

**Comment by gigglesun** [comment]
> Python 3.6 should be minimum here because of the f-strings. Did you try to install the latest release I made yesterday v0.2.1? I tested the newest release locally on my machine with Python 3.10 and it should be working okay. Pipeline is also checking for python 3.6 and it should be working fine.

yes, I just found that our system gdb use default python version 2.7 and I installed a new gdb in my home with version 12.1, but seems I cannot change the python version which gdb(12.1) use, when I configure to use a new python version with gdb, it will failed to make and compile as below:
` ./configure --with-python=/home/xxxx/tools/anaconda3/bin/python3.8 && make`
but when configure to use the system default python version as below, the gdb(12.1) can be installed successfully as below:
` ./configure --with-python=/usr/bin/python && make`
Use this gdb to work with asm2cfg will have the issue as above.


the asm2cfg python package is installed under my home folder `/home/xxxx/tools/anaconda3/lib/python3.8/site-packages/asm2cfg/`

**Comment by gigglesun** [comment]
Just sovled the python issue by not using python under anaconda3 but install a new standalone pthyon and use below command
`./configure --with-python=/home/xxx/python/3.8/bin/python && make`

I add following import location in the python file so that gdb can find the right location:



/home/xxxx/tools/anaconda3/lib/python3.8/site-packages/asm2cfg/asm2cfg.py

```
sys.path.append("/home/qsun/tools/anaconda3/lib/python3.8/site-packages")
from graphviz import Digraph
```

/home/xxxx/tools/anaconda3/bin/gdb_asm2cfg.py

```
import sys
sys.path.append("/home/qsun/tools/anaconda3/lib/python3.8/site-packages/asm2cfg")

from asm2cfg import asm2cfg
```

when start gdb, it have below error message:
```
File "/home/xxxx/tools/anaconda3/bin/gdb_asm2cfg.py", line 15, in <module>
    from asm2cfg import asm2cfg
ImportError: cannot import name 'asm2cfg' from 'asm2cfg'
```

**Comment by gigglesun** [comment]
close this issue since it's a non-standard python enviroment problem and may cannot reproduce
